spec/controllers/slack_controller_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+describe ::DiscourseSlack::SlackController do
+  routes { ::DiscourseSlack::Engine.routes }
+
+  PLUGIN_NAME = "discourse-slack-official".freeze
```

Comment:
This is duplicating the `PLUGIN_NAME` constant defined in `plugin.rb`. We can just move `PLUGIN_NAME` in `plugin.rb` under `DiscourseSlack` so that we can reference it by calling `DiscourseSlack::PLUGIN_NAME`